pkg/apiroutes/templates.go

```diff
@@ -127,7 +140,7 @@ func GetTemplateRepos() ([]utils.TemplateRepo, error) {
 
 // AddTemplateRepo adds a template repo to PFE and
 // returns the new list of existing repos
-func AddTemplateRepo(URL, description string, name string) ([]utils.TemplateRepo, error) {
+func AddTemplateRepo(URL, description string, name string, conID string) ([]utils.TemplateRepo, error) {
```

Comment:
This can be written neater as - `AddTemplateRepo(URL, description, name, conID string)`. No need to always define the data type if they are all `string`. I also notice `URL` isn't set but the rest are, thus introducing a mixture

Reply:
done